style.css

```diff
@@ -783,9 +782,8 @@ a:active {
 .site-footer .widget-area a,
 .posts-navigation a,
 .widget_authors a strong {
-	border-bottom: 2px solid #767676;
-	-webkit-transition: background-color 0.2s ease-in-out, border-color 0.2s ease-in-out, color 0.3s ease-in-out;
-	transition: background-color 0.2s ease-in-out, border-color 0.2s ease-in-out, color 0.3s ease-in-out;
+	box-shadow: inset 0 -1px 0 rgba( 15, 15, 15, 1.0);
```

Comment:
Remove the space after the opening brace and remove the trailing zero i.e this: `rgba(15, 15, 15, 1);`